README.md

```diff
@@ -241,3 +242,15 @@ ib-sriov supports the following [CNI's Capabilities / Runtime Configuration](htt
 
 EOF
 ```
+# SR-IOV Network Operator
+[SR-IOV Network Operator](https://github.com/openshift/sriov-network-operator) can manages the SR-IOV interfaces on the nodes e.g. change the number of VFs on the node, it can be used also to change the link type for the interfaces ETH to IB and vice versa, the [network policy example](https://github.com/openshift/sriov-network-operator/blob/master/deploy/crds/sriovnetwork.openshift.io_v1_sriovnetworknodepolicy_cr.yaml#L38) shows how to use the operator to change the link type for the given PCI Addresses of PFs.
+
+# System Requirements and Topology
+InfiniBand SR-IOV CNI works with kernel 5.5 which supports rdma namespace isolation and read/write GUID of the VFs.
```

Comment:
> with kernel 5.5
in RDMA CNI we require kernel 5.3 is there a reason to go with 5.5 ?

https://github.com/Mellanox/rdma-cni#overview

Reply:
What about the get Guid fix? I think it is kernel 5.5

Reply:
yes 5.5 and the VF index fixed is in 5.6 as it was proposed just after 5.5 release.

Reply:
so please update to 5.6 kernel

Reply:
i dont think we use this functionality, as we GET the hardware address and SET the port and node guid so its not critical to update to 5.6 IMO

(issue that was fixed was the netlink GET port/node GUID of VF)